ggshield/core/config/user_config.py

```diff
@@ -32,23 +41,12 @@ class UserConfig(YAMLFileConfig):
     banlisted_detectors: Set[str] = field(default_factory=set)
     ignore_default_excludes: bool = False
 
-    def update_config(self, data: Dict[str, Any]) -> bool:
-        # If data contains the old "api-url" key, turn it into an "instance" key,
-        # but only if there is no "instance" key
-        try:
-            api_url = data.pop("api_url")
-        except KeyError:
-            pass
-        else:
-            if "instance" not in data:
-                data["instance"] = api_to_dashboard_url(api_url, warn=True)
-        return super(UserConfig, self).update_config(data)
-
     def save(self, config_path: str) -> None:
```

Comment:
For my own understanding, are we writing on config files when calling save on the config?

Reply:
Yes, that's what is happening.